### ISSUE TITLE
spellright doesn't work in the `perl`  file type.

### ISSUE THREAD

**Issue by chylli** [bartosz-antosik/vscode-spellright#96]
Hi, 
This extension is great. But seems it doesn't support perl file type. Will you please add it in the future ?

**Comment by bartosz-antosik** [comment]
Could you help a little please?

By `perl` file type I understand Perl language files (.pl?) - am I right?

The extension should work fo `perl` document type with `#` as a "to the end of line" comments.

It will not work for `perl6` type of documents however as there is some inconsistency about block comments like described [here](https://docs.perl6.org/language/syntax#Comments) and `pod` comments.

The questions are:

1. What extension you use to get syntax highlighting?
2. Could you provide some example of a file with comments/strings so I could test the parser?

**Comment by chylli** [comment]
The file type is 'pm',
the common comment like

```
# helllooooooooooo wolrd 
```
is ok. but the pod comment is not:
```
=head2 function name

hellooooooo world

```


I'm using `Perl` 1.14.4 to get syntax highlighting.

Please save this code as a file like `a.pm`:
```
package hello;

=head1 descriptionnnn

Thsi is a descrptoin for Spell Right.

=cut

# liness in comment line seeeeeeeeeeeeems ok 


1;

```

**Comment by bartosz-antosik** [comment]
Thanks for helpful & complete input! Your request is done in just published version 2.0.14. Please give me some feedback, in this thread, whether it works for you.

**Comment by chylli-binary** [comment]
spellright 3.0.48
Cannot work with either perl file `.pl` or `.pm` , either `pod` doc or comment.

I haven't used vscode for long time.  So I don't know it is broken again, or didn't work at first.
